docs/root/start/install.rst

```diff
@@ -201,6 +208,18 @@ The following table shows the available Docker images
      - |DOCKER_IMAGE_TAG_NAME|
      -
      -
+   * - `envoyproxy/envoy-distroless <https://hub.docker.com/r/envoyproxy/envoy-distroless/tags/>`_
+     - Release binary with symbols stripped on top of a distroless Debain base.
+     -
+     -
+     - See Docker Hub
+     - See Docker Hub
+   * - `envoyproxy/envoy-distroless-dev <https://hub.docker.com/r/envoyproxy/envoy-distroless-dev/tags/>`_
+     - Release binary with symbols stripped on top of a distroless Debain base.
```

Comment:
move this down - so it matches existing order - ie dev images are after

also Debian is spelt incorrectly - but i think rather remove it altogether as its confusing here and not needed - so just a "distroless base"

Reply:
Got it